platform/jewel/int-ui/int-ui-standalone/build.gradle.kts

```diff
@@ -13,12 +13,12 @@ intelliJThemeGenerator {
     val targetIdeaVersion = "idea/243.19420.21"
     register("intUiLight") {
         themeClassName = "org.jetbrains.jewel.intui.core.theme.IntUiLightTheme"
-        themeFile = "platform/platform-resources/src/themes/expUI/expUI_light.theme.json"
+        themeFile = "../../platform/platform-resources/src/themes/expUI/expUI_light.theme.json"
         ideaVersion = targetIdeaVersion
     }
     register("intUiDark") {
         themeClassName = "org.jetbrains.jewel.intui.core.theme.IntUiDarkTheme"
-        themeFile = "platform/platform-resources/src/themes/expUI/expUI_dark.theme.json"
+        themeFile = "../../platform/platform-resources/src/themes/expUI/expUI_dark.theme.json"
         ideaVersion = targetIdeaVersion
```

Comment:
Nit: If we keep the idea version, we should also get it automatically from the project. The var is only used to create a header comment

<img width="1435" alt="image" src="https://github.com/user-attachments/assets/c2495b87-156f-41da-a666-69db866eac8f" />

Reply:
We could probably remove the source and version. As we get from the local build path, we don't need that info anymore